sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueAsyncClient.java

```diff
@@ -20,13 +20,13 @@
 import com.azure.storage.queue.implementation.AzureQueueStorageImpl;
 import com.azure.storage.queue.models.DequeuedMessage;
 import com.azure.storage.queue.models.EnqueuedMessage;
-import com.azure.storage.queue.models.MessageIdUpdateHeaders;
-import com.azure.storage.queue.models.MessageIdsUpdateResponse;
+import com.azure.storage.queue.implementation.MessageIdUpdateHeaders;
```

Comment:
Do we want the model classes to just move to `implementation` or `models.implementation`?

Reply:
@JonathanGiles

Reply:
Or `implementation.models`